.github/workflows/quarkus-nightly.yml

```diff
@@ -0,0 +1,78 @@
+name: Quarkus Nightly
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - quarkus-nightly
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  # Only cancel jobs for PR updates
+  group: quarkus-nightly-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  rebase-quarkus-nightly-branch:
+    name: Rebase quarkus-nightly branch
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: quarkus-nightly
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config --global user.name "GitHub Actions"
```

Comment:
Not sure if this works or not, but other places we set this a bit differently. See:
https://github.com/keycloak-rel/keycloak-rel/blob/72a3b5ce0015e0ae5f5d04d0b9e17040a390d733/.github/workflows/x-create-tags.yml#L52-L53

Reply:
It worked, but I unified it as suggested.